CHANGELOG.md

```diff
@@ -1,37 +1,62 @@
 ## CHANGELOG
 
+### 07/23/21
+
+#### Updated
+
+- Added a message to be displayed if JS/CSS files cannot be found.
```

Comment:
"...cannot be found when the app is running and the`/admin` page is visited."

Reply:
Fixed!